pennylane/plugins/default_qubit.py

```diff
@@ -345,35 +345,44 @@ def mat_vec_product(self, mat, vec, wires):
         return np.reshape(state_multi_index, 2 ** self.num_wires)
 
     def expval(self, observable, wires, par):
-        r"""Expectation value of observable on specified wires.
```

Comment:
how come docstring was deleted? If anything, it should be updated to clarify difference between exact and estimated expectation value calculations

Reply:
Because it will be automatically inherited from the base class, @josh146 requested this.

Reply:
:+1: In the case where the docstring are near identical, it is always best to inherit the docstring from the base class method. 

However, the one exception is if the overwritten method contains new functionality or behaviour which should be documented.